dist/weather-card.js

```diff
@@ -185,12 +185,19 @@ class WeatherCard extends LitElement {
 
   renderDetails(stateObj) {
     const sun = this.hass.states["sun.sun"];
+    const lang = this.hass.selectedLanguage || this.hass.language;
```

Comment:
Since we now use this in multiple places (line 250) we should create the const in the main render and pass it to the render functions.